wcfsetup/setup/db/install.sql

```diff
@@ -1255,6 +1255,17 @@ CREATE TABLE wcf1_search_keyword (
 	KEY (searches, lastSearchTime)
 );
 
+DROP TABLE IF EXISTS wcf1_service_worker;
+CREATE TABLE wcf1_service_worker (
+	workerID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	userID INT(10) NOT NULL,
+	endpoint TEXT NOT NULL,
+	publicKey VARCHAR(88) DEFAULT '' NOT NULL,
+	authToken VARCHAR(24) DEFAULT '' NOT NULL,
```

Comment:
```suggestion
	publicKey VARCHAR(88) NOT NULL,
	authToken VARCHAR(24) NOT NULL,
```

That default value makes no sense, these fields must never be empty.